lib/ovirt/inventory.rb

```diff
@@ -130,7 +130,7 @@ def standard_collection(uri_suffix, element_name = nil, paginate = false, sort_b
     SECONDARY_ITEMS = {
       # Key          RHEVM API methods
       :datacenter => [:storagedomains],
-      :host       => [:statistics, :nics], # :cdroms, tags
+      :host       => [:statistics, :host_nics], # :cdroms, tags
```

Comment:
This is exactly what I though and I changed it together with storage domain refresh for a vm. This change broke the refresh so reverting it now.